src/index.js

```diff
@@ -20,20 +16,49 @@ import 'regenerator-runtime/runtime';
 
 // CSS imports
 import './css/roar.css';
+import './css/custom.css';
 
 // Local modules
 import { initConfig, initRoarJsPsych, initRoarTimeline } from './config';
 import {
   NUM_BLOCKS,
-  NUM_TRIALS,
+  ADAPTIVE_NUM_TRIALS,
+  PRACTICE_NUM_TRIALS,
   CONTROL_NUM_TRIALS,
   SHOW_CONTROL_TRIALS,
   STIMULUS_FONT_SIZE,
+  VIDEO_HEIGHT,
+  VIDEO_WIDTH,
   IGNORE_CASE,
+  CORRECT_KEY_PRESS,
+  CORRECT_KEY_TEXT,
+  WRONG_KEY_PRESS,
+  WRONG_KEY_TEXT,
 } from './utils';
 
 // assets
-import intro_video from '../assets/intro-video.mp4';
+import intro_video_1 from '../assets/intro-vid-pt-1.mp4';
+import intro_video_2 from '../assets/intro-vid-pt-2.mp4';
+import intro_video_3 from '../assets/intro-vid-pt-3.mp4';
+import game_instructions_1 from '../assets/game-instructions-1.mp4';
+import game_instructions_2 from '../assets/game-instructions-2.mp4';
+import game_instructions_3 from '../assets/game-instructions-3.mp4';
+import pos_instuctions_feedback from '../assets/instruction-fb-pos.mp4';
+import neg_instuctions_feedback from '../assets/instructions-fb-neg.mp4';
+import two_back_instructions from '../assets/two-back-instructions.mp4';
+import three_back_instructions from '../assets/three-back-instructions.mp4';
+import fix_robot_1 from '../assets/fix-robot-1.mp4';
+import fix_robot_2 from '../assets/fix-robot-2.mp4';
+import fix_robot_3 from '../assets/fix-robot-3.mp4';
+import fix_robot_4 from '../assets/fix-robot-4.mp4';
+import fix_robot_5 from '../assets/fix-robot-5.mp4';
+import fix_robot_6 from '../assets/fix-robot-6.mp4';
+import fix_robot_n from '../assets/fix-robot-n.mp4';
+import right_arrow_image from '../assets/right-trial-screen-arrow.png';
+import left_arrow_image from '../assets/left-trial-screen-arrow.png';
+import end_video from '../assets/end-video.mp4';
+import generic_game_break from '../assets/game-break.mp4';
```

Comment:
I like the name `generic_game_break`, good job switching it over from just `game_break`. Definitely more readable.